src/shared/consts.tsx

```diff
@@ -333,3 +333,8 @@ export const JEST_TEST_TIMEOUT = 10000;
 export const NON_UNIQUE_VALUE_MESSAGE = 'Non-unique value.';
 
 export const NULL_ANSWER = 'value: null';
+
+// TODO: temporally implementation of LaunchDarkly.
```

Comment:
```suggestion
// TODO: temporary implementation of LaunchDarkly. Will be removed when LD integration is added
```
and perhaps it would be good to also include a link to the JIRA ticket for the LD integration task?